compose/service.py

```diff
@@ -1283,12 +1284,36 @@ def merge_volume_bindings(volumes, tmpfs, previous_container, mounts):
     return list(volume_bindings.values()), affinity
 
 
+def try_get_image_volumes(container):
+    """
+        Try to get the volumes from the existing container. If the image does
+        not exist, prompt the user to either continue (rebuild the image from
+        scratch) or throw an exception.
+    """
+
+    try:
+        image_volumes = [
+            VolumeSpec.parse(volume)
+            for volume in
+            container.image_config['ContainerConfig'].get('Volumes') or {}
+        ]
+        return image_volumes
+    except ImageNotFound:
+        prompt = \
+            "WARNING: Image not found. Continue by rebuilding the image? [yN]"
```

Comment:
Ideally we would be a little more verbose here, to warn the user about the risk of data loss and recommend they back up  if necessary. The current message doesn't convey why someone wouldn't want to rebuild.